### PR TITLE
Uri/post first run fixes

### PR DESCRIPTION
**Pull request by urikirsh**
Answering a few questions that were asked during the first live lecture in the README. 
Also small editing touches.
James, I think the tutorial is very clear now, excellent work!

### CODE REVIEW

File: Tutorials/Lesson1/README.md [hunk 1]
```diff
@@ -26,14 +26,20 @@ Thinking about the function `deposit`, a basic property is:
...
-Additionally, all possible calling contexts are taken into account. Certora prover represents the calling context through the struct variable `env`. Declaring a single `env` variable suffices to capture all aspects of the calling contexts, but they can also be addressed individually.
+All possible inputs to the deposit function (all possible deposited amounts) are taken into account. 
+Additionally, all possible calling contexts are considered. Certora prover represents the calling context through the struct variable `env`. Declaring a single `env` variable suffices to capture all aspects of the calling contexts, but they can also be addressed individually.
 Some example aspects of the calling context are:
- - "who is the depositor?" (`env.msg.sender`)
- - "what was the initial balance of the depositor?" (`env.msg.sender.balance`)
- - "in which block does the deposit occur?" (`env.block.number`)
- - "at which time does the deposit occur?" (`env.block.timestamp`)
+ - "Who is the depositor?" (`env.msg.sender`)
+ - "What was the initial balance of the depositor?" (`env.msg.sender.balance`)
```
Comment (nd-certora) [inline Tutorials/Lesson1/README.md]: env.msg.sender.balance  is a nice feature to add  but still not supported. Let's remove for now

File: Tutorials/Lesson1/README.md [hunk 2]
```diff
@@ -26,14 +26,20 @@ Thinking about the function `deposit`, a basic property is:
...
+ - "In which block does the deposit occur?" (`env.block.number`)
+ - "At which time does the deposit occur?" (`env.block.timestamp`)
  - (and many more)
+ 
+Notice that each function must get an `env` variable as its first parameter:
+`deposit(e, amount);`.
+
+By default, the Certora prover will ignore all paths where any function call reverts, and they will not be considered violations of the rule.
```
Comment (nd-certora) [inline Tutorials/Lesson1/README.md:40]: "any function" shouldn't it be "the function"?

**Review by nd-certora — APPROVED** [review]
I few comments

**Review by wilcoxjay — APPROVED** [review]
thanks for the edits, Uri! and for the record, Nurit created all the materials!

**Pull request merged**